packages/components/nodes/chatmodels/ChatIBMWatsonx/ChatIBMWatsonx.ts

```diff
@@ -75,6 +75,59 @@ class ChatIBMWatsonx_ChatModels implements INode {
                 step: 1,
                 optional: true,
                 additionalParams: true
+            },
```

Comment:
thanks @eduardconstantin , can you increment the version of this node?

Reply:
sure, I change it to 2.0